frontend/public/components/nav.jsx

```diff
@@ -392,6 +392,8 @@ export class Nav extends React.Component {
             <ResourceNSLink resource="secrets" name="Secrets" onClick={this.close} />
             <ResourceNSLink resource="configmaps" name="Config Maps" onClick={this.close} />
             <Sep />
+            <ResourceNSLink resource="serviceinstances" name="Service Instances" onClick={this.close} required={FLAGS.SERVICE_CATALOG_AVAILABLE} />
+            <ResourceNSLink resource="servicebindings" name="Service Bindings" onClick={this.close} required={FLAGS.SERVICE_CATALOG_AVAILABLE} />
```

Comment:
Let's go ahead and create a "Service Catalog" category and hide or show the category based on the flag